actions.yaml

```
@@ -62,4 +62,14 @@ restore:
       description: |
         A backup-id to identify the backup to restore. Format: backup-id=<string>.
   required:
-    - backup-id
\ No newline at end of file
+    - backup-id
+
+resume-upgrade:
+  description: Upgrade remaining units (after you manually verified that upgraded units are healthy).
+force-upgrade:
+  description: Force upgrade of this unit.
```

I think we need to be more clear, e.g.:
```suggestion
  description: |
        WARNING: running force-upgrade may result in breaking the cluster or data loss. Evaluate very carefully before executing it.
        Force upgrade of this unit. Use this action to override the upgrade constraints, e.g. version mismatch errors.
```